newlang/hckernel/mlir/lib/Dialect/PyIR/IR/PyIROps.cpp

```diff
@@ -143,6 +198,11 @@ static void printArgList(mlir::OpAsmPrinter &printer, Op /*op*/,
   printer << ')';
 }
 
+void hc::py_ir::PyFuncOp::getCanonicalizationPatterns(
```

Comment:
move near to the rest of `PyFuncOp` funcs

Reply:
done